xml/obs/devel:LEO.xml

```diff
@@ -0,0 +1,26 @@
+<openQA
+    project_pattern="devel:LEO"
+	dist_path="obspublish-other::openqa/devel:LEO/images/x86_64"
```

Comment:
I think it is safe to replace `x86_64` with `*` here if we target several architectures, e.g. openSUSE:Leap:15.2:MicroOS.xml does the same.